gradle/java.gradle

```diff
@@ -6,10 +6,6 @@ apply plugin: 'com.palantir.baseline-idea'
 
 sourceCompatibility = 1.8
 
-checkstyle {
-    configDir = rootProject.file(".baseline")
-}
-
-if (System.env.CIRCLE_TEST_REPORTS) {
-    test.reports.junitXml.destination = new File(System.env.CIRCLE_TEST_REPORTS, project.getName())
+if (System.env.CI) {
```

Comment:
I prefer to have something like the following in the circle config.yml so the magic path only shows up in one place 
```
       - run:
          command: |
            mkdir -p ~/junit
            find . -type f -regex ".*/build/.*TEST.*xml" -exec cp {} ~/junit/ \;
          when: always
      - store_test_results:
          path: ~/junit 
```